symfony/security-bundle/7.3/config/packages/security.yaml

```diff
@@ -0,0 +1,39 @@
+security:
+    # https://symfony.com/doc/current/security.html#registering-the-user-hashing-passwords
+    password_hashers:
+        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
+    # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
+    providers:
+        users_in_memory: { memory: null }
+    firewalls:
+        dev: # Exception for the web debug toolbar and assets (only needed if you have a broad `access_control`):
```

Comment:
I'd tweak these comments and its formatting too. Instead of a single long line, we could add the comment above the related config:

```yaml
firewalls:
    # Disable security for dev tools and static assets (only needed if access_control is broad)
    dev:
        # 'assets/' is for AssetMapper, 'build/' for Webpack Encore
        # (note: no regex delimiters needed; Symfony adds them automatically)
        pattern: ^/(_profiler|_wdt|assets|build)
```